spec/setup_acceptance_node.pp

```diff
@@ -0,0 +1,54 @@
+class { 'foreman::repo':
+  repo => 'nightly',
+}
+
+stdlib::ensure_packages(['podman'])
+
+$container_file = "[Unit]
+Description=Advisor Engine
+After=network-online.target
+[Container]
+Image=registry.access.redhat.com/ubi9/python-312
+Exec=python /usr/local/bin/python-https-server
```

Comment:
This is obviously still a fake, but a much more elaborate one :D

We should be able to replace this with the "real" deal once a build is available.